### DISCUSSION TITLE
SqlSyncProvider or SqlSyncChangeTrackingProvider

### DISCUSSION THREAD

**Discussion by vunder** [Mimetis/Dotmim.Sync#644] [Q&A]
Hi
Is there some test results of using tracking or non-tracking Microsoft SQL Server Provider?
There are several questions regarding that: what is faster, what is better (memory/cpu), what are pitfalls?

**Comment by Mimetis** [comment]
Nop.
Change tracking is handled by sql server  itself.
So i guess it has better performances.
If you have any idea on how to make such comparaison, let me know

**Reply by vunder** [reply]
Ok. I'll try to perform some local results
